cloudant-client/src/test/java/com/cloudant/tests/DocumentsCRUDTest.java

```diff
@@ -93,13 +74,14 @@ public void findJsonObject() {
         assertNotNull(jsonObject);
     }
 
+    /*
     @Test
```

Comment:
Is this commented because the test is failing? If so, should we instead use `@Disabled`?

Reply:
Fixed in 3e9d231